app.json

```diff
@@ -59,6 +59,18 @@
     },
     "RAILS_ENV": {
       "required": false
+    },
+    "OSEM_TWITTER_CONSUMER_KEY": {
+      "required": true
```

Comment:
It's not mandatory. If the conference doesn't have a twitter account, twitter partial will not display.